src/include/platform/PlatformManager.h

```diff
@@ -183,24 +201,18 @@ class PlatformManager
      * returns.
      */
     CHIP_ERROR StopEventLoopTask();
-    void LockChipStack();
-    bool TryLockChipStack();
-    void UnlockChipStack();
-    void Shutdown();
 
-#if CHIP_STACK_LOCK_TRACKING_ENABLED
-    bool IsChipStackLockedByCurrentThread() const;
+    // These functions should be gated by CHIP_DEVICE_CONFIG_ENABLE_BG_EVENT_PROCESSING
+    // but the PlatformManagerImpl header (included below) makes that tricky
+    // so always declare them here even if the are not defined by the platform implementation.
+#if 1 || defined(CHIP_DEVICE_CONFIG_ENABLE_BG_EVENT_PROCESSING) && CHIP_DEVICE_CONFIG_ENABLE_BG_EVENT_PROCESSING
```

Comment:
Is `#if 1` intendend? This pulls it in always.

Reply:
Intended, as per comment. I can clarify the comment that the #if 1 is how they are being declared regardless.

Reply:
Actually this is no longer relevant as the conditional compilation here has been entirely removed.